pkgs/build-support/rust/hooks/default.nix

```
@@ -126,8 +127,8 @@ in {
     bindgenHook = callPackage ({}: makeSetupHook {
```

If anyone wants to try this again later, the real problem is with libcxx; the cross-compilation issue can be easily fixed by adding `rustc` to `bindgenHook`'s `callPackage` arguments.

Arguably, it would make sense to make that change here as well, since it makes it possible to override the clang version used by `bindgenHook`, but I'd be fine keeping this as a pure revert. In this case the change would look like:
```suggestion
    bindgenHook = callPackage ({ clang }: makeSetupHook {
```